File: app/src/main/java/me/zipi/navitotesla/api/GithubApi.java
package me.zipi.navitotesla.api;

import java.util.List;

import me.zipi.navitotesla.model.Github;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface GithubApi {

    @GET("/repos/{owner}/{repo}/releases?per_page=10")
    Call<List<Github.Release>> getReleases(@Path("owner") String owner, @Path("repo") String repo);

}

File: app/src/main/kotlin/me/zipi/navitotesla/api/GithubApi.kt
package me.zipi.navitotesla.api

import me.zipi.navitotesla.model.Github.Release
import retrofit2.Response
import retrofit2.http.GET
import retrofit2.http.Path

interface GithubApi {
    @GET("/repos/{owner}/{repo}/releases?per_page=10")
    suspend fun getReleases(
        @Path("owner") owner: String,
        @Path("repo") repo: String
    ): Response<List<Release>>
}